tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sqlite_data.py

```diff
@@ -0,0 +1,74 @@
+"""
+To run this code as a local test, use the following console command:
+```
+pytest -v --docs-tests -m integration -k "how_to_connect_to_sqlite_data" tests/integration/test_script_runner.py
+```
+"""
+
+import pathlib
+import great_expectations as gx
+
+sqlite_database_path = str(
+    pathlib.Path(
+        gx.__file__,
+        "..",
+        "..",
+        "tests",
+        "test_sets",
+        "taxi_yellow_tripdata_samples",
+        "sqlite",
+        "yellow_tripdata.db",
+    ).resolve(strict=True)
+)
+
+my_table_name = "yellow_tripdata_sample_2019_01"
+
+
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sqlite_data.py import">
```

Comment:
@TrangPham Non-blocking: since we are not using this "import" snippet, we can remove it (maybe in a future refactor we will replace the component tag containing the old-style code with something like this "import").  Thanks.